src/main.rs

```diff
@@ -233,6 +234,16 @@ fn main() {
         let delta = (diff.num_nanoseconds().unwrap() as f64) / frame_time;
         let (width, height) = window.drawable_size();
 
+        let fps_cap = {
+            let console = game.console.lock().unwrap();
+            let vsync_changed = console.get(settings::R_VSYNC).clone();
+            if vsync != vsync_changed {
+                vsync = vsync_changed;
+                sdl_video.gl_set_swap_interval(if vsync { 1 } else { 0 });
+            }
+            console.get(settings::R_MAX_FPS).clone()
```

Comment:
I tend to prefer `*console.get(settings::R_MAX_FPS)` vs cloning when it comes to copy types